### ISSUE TITLE
Unable to downlad word list from Market place

### ISSUE THREAD

**Issue by shadowbone94** [adityakamble49/word-list#4]
Exactly what the title says. I'm unable to downlad any sets from Market place as it just keeps loading. Any advice please ?

**Comment by adityakamble49** [comment]
Yes. Actually I have changed Marketplace structure to suite new version under `develop` branch. That't why its broken for master branch version.
New version is still under development. I will revert Marketplace to suit old version so it can be used with marketplace.
I will let you know in this issue thread once its done so you can try again

**Comment by shadowbone94** [comment]
@adityakamble49 That was quick :smile:
I appreciate your help.

**Comment by adityakamble49** [comment]
Hi @shadowbone94 

I have updated Word List Marketplace to support Android app
Uninstall and Reinstall android app and try again.
Let me know if it works on your side.

Meanwhile I am migrating all list to old format

**Comment by shadowbone94** [comment]
@adityakamble49 Just checked and boy it worked ! Thank you.
Might I ask just one thing? Where is the list from Market place loaded from? Because I think there are one or two missing from the previous version. So I was wondering if I could download the list of words from the server( if there is any) externally via browser, and then import it to the app. As such there is an option to import export list.
Thanks,

**Comment by adityakamble49** [comment]
@shadowbone94 Marketplace is loaded from [WordList-Marketplace](https://github.com/adityakamble49/wordlist-marketplace/tree/master/wordlists) Repository
Actually I had converted all Word Lists to new format for v2.0. As you pointed out it stopped supporting old version. So I ported few of the lists to old version so people can use it till new version comes out.

Still I am migrating all list. All lists will be available soon on older versions too
